webapp/src/main/webapp/templates/freemarker/body/partials/individual/propStatement-dataDefault.ftl

```diff
@@ -1,4 +1,4 @@
-<#-- $This file is distributed under the terms of the license in LICENSE$ -->
+<#-- $This file is distributed under the terms of the license in /doc/license.txt$ -->
```

Comment:
```suggestion
<#-- $This file is distributed under the terms of the license in LICENSE$ -->
```
Undo reversion to old license location

Reply:
Updated pull request to remove the propStatement-dataDefault.ftl. This is in Vitro with Vitro pull 101